alacritty_terminal/src/window.rs

```diff
@@ -12,11 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 use std::convert::From;
-#[cfg(not(any(target_os = "macos", target_os = "windows")))]
+use std::process;
```

Comment:
this import is useless on macOS/Windows so we need the same `#cfg` as for `c_void`. This one should be the last thing from my side.

Reply:
Should be sorted.  Thanks again.